test/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -239,8 +239,15 @@ namespace {this.Name}
     public class {this.Name}Class
     {{
         public static string Name {{ get {{ return ""{this.Name}""; }} }}
-    }}
-}}";
+";
+                    foreach (var dependency in this.ReferencedProjects)
+                    {
+                        source += $"        public static string {dependency.Name} {{ get {{ return {dependency.Name}; }} }}" + Environment.NewLine;
```

Comment:
This is a reference to the supporting projects - I did get it wrong; fixed now:
**{dependency.Name}.{dependency.Name}Class.Name;**